util/libcrypto.num

```diff
@@ -4572,3 +4572,5 @@ EC_POINT_set_affine_coordinates         4525	1_1_1	EXIST::FUNCTION:EC
 EC_POINT_get_affine_coordinates         4526	1_1_1	EXIST::FUNCTION:EC
 EC_GROUP_set_curve                      4527	1_1_1	EXIST::FUNCTION:EC
 EC_GROUP_get_curve                      4528	1_1_1	EXIST::FUNCTION:EC
+OCSP_resp_get0_tbs_sigalg               4529	1_1_1	EXIST::FUNCTION:OCSP
+OCSP_resp_get0_respdata                 4530	1_1_1	EXIST::FUNCTION:OCSP
```

Comment:
It should be 1_1_0j instead of 1_1_1 if you aim to target the 1.1.0 branch.
See the `1_1_0i` for the previous release of this branch, that is present 14 lines above.

Reply:
Is that a second PR targeting the 1_1_0 branch or is it okay to commit 1_1_0j for the 1_1_1 branch as well?

Reply:
@reaperhulk the latter: Just edit the file and replace 1_1_1 with 1_1_0j.